src/data/mainnet/tokens-info.json

```diff
@@ -56,8 +56,8 @@
     "address": "0x173234922eb27d5138c5e481be9df5261faed450",
     "decimals": 18,
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/SOL.png",
-    "name": "SOL",
-    "symbol": "SOL",
+    "name": "Allbridge Wrapped SOL",
+    "symbol": "WSOL",
```

Comment:
The "W" in "WSOL" for the symbol might be confusing too because then symbol won't match the `symbol` in the erc-20 contract.  Have we discussed the pros and cons of doing something like this?

Reply:
I am OK with keeping it as SOL as long as the name clarifies that it is wrapped

Reply:
(separately, it's confusing that wrapped assets have no naming standards. cSOL, WSOL, SOL. but i have no answers, only gripes)